src/read_next/read_next.jsx

```diff
@@ -16,29 +19,78 @@ const ReadNext = () => {
                 setPosts({ posts: postsWithoutCurrent });
             });
     }, [params]);
+    const PerPageChange = (value) => {
```

Comment:
conventionally [camelCase](https://en.wikipedia.org/wiki/Camel_case) is used in Javascript for variable naming. PascalCase is only used for class names. fix here and everywhere else